lib/capybara_discoball/runner.rb

```diff
@@ -11,11 +10,15 @@ def initialize(server_factory, &block)
 
       def boot
         with_webrick_runner do
-          @server = Capybara::Server.new(@server_factory.new)
+          @server = Server.new(@server_factory.new)
           @server.boot
         end
 
-        @after_server.call(@server)
+        if @after_server
```

Comment:
You don't have to pass a block at all the Runner. Is there a nicer syntax to skip this call if there is no block?